GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationViewController.swift

```diff
@@ -30,26 +25,28 @@ extension SecureConversations {
 
         override func loadView() {
             super.loadView()
-            renderProps()
-        }
+            let hostingController: UIHostingController<ConfirmationViewSwiftUI>
+            let confirmationView = ConfirmationViewSwiftUI(
+                model: model,
+                orientation: orientation
+            )
 
-        func renderProps() {
-            let confirmationView: ConfirmationView
-            if let currentView = view as? ConfirmationView {
-                confirmationView = currentView
-            } else {
-                confirmationView = viewFactory.makeSecureConversationsConfirmationView(
-                    props: props.confirmationViewProps
-                )
-                view = confirmationView
-            }
-            confirmationView.props = props.confirmationViewProps
-        }
-    }
-}
+            hostingController = UIHostingController(rootView: confirmationView)
+            addChild(hostingController)
+            view.addSubview(hostingController.view)
+            hostingController.didMove(toParent: self)
 
-extension SecureConversations.ConfirmationViewController {
-    struct Props: Equatable {
-        let confirmationViewProps: SecureConversations.ConfirmationView.Props
+            hostingController.rootView = ConfirmationViewSwiftUI(
```

Comment:
Looks like you applied `rootView` twice. First one is on 34th line

Reply:
fixed [58bba1e](https://github.com/salemove/ios-sdk-widgets/pull/769/commits/58bba1e8ab82b1aade878890175391a5717c8375)